src/client/components/PasswordForm.tsx

```diff
@@ -190,7 +194,12 @@ const isBreached = AwesomeDebouncePromise(
   300,
 );
 
-export const PasswordForm = ({ submitUrl, fieldErrors }: Props) => {
+export const PasswordForm = ({
+  submitUrl,
+  fieldErrors,
+  submitButtonText,
+  gridAutoRow,
+}: Props) => {
```

Comment:
small question on the naming of this Props type - I've seen a pattern where interfaces for functional components are called `IComponentNameProps`, e.g: (`IPasswordFormProps`), might be a bit more descriptive, especially if it's exported at some point. I realise that wasn't added as part of this, but maybe worth considering for our components?

Reply:
There isn't a hard rule in gateway as to how we name prop interfaces, in some components you'll notice `interface ComponentProps` in others it's just `interface Props`.

My general opinion is that because props are self-contained within the component/component file, we don't need to explicitly name them, as it's easy to tell which component those props belong too.

As for prefixing interfaces with `I` there is a big debate around these, and the general consensus is that the prefix isn't necessary: https://github.com/microsoft/TypeScript-Handbook/issues/121#issuecomment-412810455

Reply:
makes sense, thanks for explaining rationale. not seen that thread before will take a look